File: Lista02/multimetro.py
import random
valores = []
soma = 0
desvio = 0

for x in range(10):
    v = random.random()
    valores.append(v)
    soma+=v

media = soma / len(valores)

print(f"\n\n--- Relatório ---")
for x,v in enumerate(valores):
    print(f"Medição {x+1} : {v:.4f}")
    desvio+= (v-media)**2
print(f"Média das medições: {media:.4f}")
print(f"Desvio padrão: {desvio:.4f}")
msg = "OK" if(desvio > (media*0.1)) else "COM PROBLEMAS"
print("Status do aparelho:", msg)

File: lista1/src/Multimetro.java
import java.util.Random;

public class Multimetro {
    public static void main(String[] args) {
        Random rand = new Random();
        float[] valores = new float[10];
        float soma = 0, media = 0, desvio = 0;
        
        for (int i = 0; i < valores.length; i++) {
            valores[i] = rand.nextFloat();
            soma += valores[i];
        }

        media = soma / valores.length;
        
        for (int i = 0; i < valores.length; i++) {
            desvio += Math.pow(valores[i] - media, 2);
        }
        desvio = (float) Math.sqrt(desvio / (valores.length - 1));

        limpaTela();
        System.out.println("--- Relatório ---");
        for(int i=0; i<valores.length; i++){
            System.out.println("Medição "+ (i+1) +": " + valores[i]);
        }
        System.out.println("Média das medições: "+ media);
        System.out.println("Desvio padrão: " + desvio);
        String msg =  desvio > (media*0.1)?"COM PROBLEMAS":"OK";
        System.out.println("Status do aparelho: "+msg);
    }

    public static void limpaTela(){
        for (short i = 1; i<100; i++) System.out.println("\n");
    }
}